kuksa_databroker/databroker/tests/read_write_values.rs

```diff
@@ -244,6 +244,24 @@ fn assert_request_failure(w: &mut DataBrokerWorld, expected_status_code: i32) {
     w.assert_status_has_code(expected_status_code)
 }
 
+#[then(expr = "the current value for {word} can not be accessed because we are unauthorized")]
```

Comment:
No word is not used as of now. Would it be okay to use it like this:
```
the request fails because the client is unauthorized
```
I agree that your option would be good but since the error would be a grpc error code I think it is nicer to name it (do not know if anybody is familiar with grpc codes. For the standard error codes used in the other examples I think everyone knows what they mean.) Otherwise yeah we could do it like you suggested. What do you think?

Reply:
Ah, but I see now there is a scenario from you which checks for status codes as well and you linked documentation to it in code. Then I could use this. But as I mentioned I think for clarity maybe it would be best to name it in theses cases. Let me know what your thoughts are :)

Reply:
FMPOV it is ok to refer to the gRPC codes since we are testing a gRPC API here ...